package.json

```diff
@@ -51,6 +51,7 @@
     "papaparse": "^4.3.7",
     "pg": "^7.4.1",
     "swagger-ui-express": "^2.0.15",
+    "underscore": "^1.8.3",
```

Comment:
are you using underscore ?

Reply:
oh no sorry i think that was just to try something out

Reply:
then remove it please ;)